<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableSetting</class>
 <widget class="QDialog" name="TableSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройка таблиц</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>389</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Таблица:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Диагноз</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Окраска</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Увеличение</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Тип клетки</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Группа признаков</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Подгруппа признаков</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Признак</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Данные таблицы:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Подгруппа признаков:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_3"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Признак:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_4"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
